solutions/USACO open19-milk-factory.mdx

```diff
@@ -0,0 +1,86 @@
+---
+id: usaco-940
+title: USACO 2019 December Bronze - Milk Factory
+author: Jesse Choe
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_factory_bronze_open19.html)
+
+The solution provided by USACO can be solved more simply and efficiently using the Depth-First Search Algorithm (DFS). Since you are likely unfamiliar with what DFS is, please refer to this link to learn more about DFS: https://usaco.guide/silver/dfs.
```

Comment:
> The solution provided by USACO can be solved more simply and efficiently using the Depth-First Search Algorithm (DFS).

???

Reply:
>  Since you are likely unfamiliar with what DFS is, please refer to this link to learn more about DFS: https://usaco.guide/silver/dfs.

Delete this sentence and just link DFS in the first sentence, like so:

```
[Depth-First Search Algorithm](/silver/dfs)
```